packages/nestjs/README.md

```diff
@@ -24,6 +24,8 @@ yarn add @sentry/nestjs
 
 ## Usage
 
+Add a instrument.ts file:
+
 ```typescript
 // CJS Syntax
 const Sentry = require('@sentry/nestjs');
```

Comment:
I would probably get rid of the syntax comments and the CJS here. Doesn't make sense for nestjs I think.